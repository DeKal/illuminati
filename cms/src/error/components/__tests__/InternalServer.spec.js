import * as React from 'react'
import { shallow } from 'enzyme'
import InternalServer from '../InternalServer'

describe('InternalServer', () => {
  it('should render internal server error page correctly', function() {
    const wrap = shallow(<InternalServer />)
    expect(wrap).toMatchSnapshot()
  })
})
