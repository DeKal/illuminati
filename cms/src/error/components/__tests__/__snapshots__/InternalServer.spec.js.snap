// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InternalServer should render internal server error page correctly 1`] = `
<Layout
  description="The server encountered an internal error or misconfiguration and was unable to complete your request."
  title="An error 500 Internal Server Error occurred"
/>
`;
